ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -469,6 +470,7 @@ def add_capacity(osd_size_capacity_requested):
         device_sets_required + old_storage_devices_sets_count
     )
     lvpresent = localstorage.check_local_volume()
+    lvspresent = localstorage.check_local_volume_set()
```

Comment:
maybe change `lvspresent` to `lv_set_present` as `lvs` could be mistakenly translated as local volumes and not local volume set

Reply:
Done